prime_sieve.py

```
@@ -0,0 +1,66 @@
+            for multiple in range(start, high + 1, p):
+                sieve[multiple - low] = 0
+        for i in range(size):
+            if sieve[i]:
+                yield low + i
+        low = high + 1
+        high = low + segment_size - 1
+
+
+if __name__ == "__main__":
+    import argparse
+    parser = argparse.ArgumentParser(description="Generate primes up to a limit using a segmented sieve.")
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)